packages/wallet-ui/src/components/ui/organism/UpgradeModel/UpgradeModel.view.tsx

```diff
@@ -117,8 +117,9 @@ export const UpgradeModelView = ({ address }: Props) => {
               {shortenAddress(txnHash)}{' '}
             </Txnlink>
             <br />
-            Unfortunately, you reached the maximum number of upgrade tentatives
-            allowed.
+            Your upgrade/deploy transaction is still pending and has reached the
+            maximum retry limit for status checks. Please wait for the
+            transaction to complete.
```

Comment:
Could we add the link to the transaction on the block explorer ?

Reply:
you already have a link above